app/javascript/components/assessment/steps/SymptomsAssessment.js

```diff
@@ -47,6 +53,7 @@ class SymptomsAssessment extends React.Component {
   }
 
   navigate() {
+    this.setState({ loading: true });
     this.props.submit();
```

Comment:
This should be in the new setState callback to guarantee it happens after.

Reply:
done